src/Helper/Dsn.php

```diff
@@ -42,7 +42,7 @@ public function getDsn(): string
         $this->dsn = "$this->driver:" . "host=$this->host" . ';' . "dbname=$this->dbname";
```

Comment:
The Dsn itself isn't a helper but a value object and belongs to `Yiisoft\Db\Connection` namespace.

Reply:
Fixed.